<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogVLM_grib_ui</class>
 <widget class="QDialog" name="DialogVLM_grib_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>258</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VLM grib download</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>238</width>
        <height>166</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="radio1">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio2">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio3">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio4">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radio5">
         <property name="text">
          <string>No grib file</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>Download</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>DialogVLM_grib_ui</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>162</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>119</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>DialogVLM_grib_ui</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>270</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>357</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
